src/components/HorizontalScrollNav.jsx

```diff
@@ -0,0 +1,71 @@
+import React,{useContext} from 'react';
+import { ScrollMenu, VisibilityContext } from 'react-horizontal-scrolling-menu';
+import Movie from './movies/Movie';
+import styled from 'styled-components';
+
+const ContainerItem = styled.div`
+  width:200px;
+  margin-right:15px;
+`;
+
+const LeftButton = styled.button`
+  margin-right:10px;
+  background: transparent;
+  border:0px;
+  position:absolute;
+  left:0px;
+  z-index:9999;
+`;
+
+const RightButton = styled.button`
+  margin-left:10px;
+  background: transparent;
+  border:0px;
+  position:absolute;
+  right:0px;
+  z-index:9999;
+  background-color:#CCCCCC;
+  border-radius:10px;
+  width:30px;
+  height:30px;
+`;
```

Comment:
These styled components should be defined in a separate file.
Maybe the component can be placed in a folder with the following structure:

```
src/
--components/
----HorizontalScrollNav/
------HorizontalScrollNav.jsx
------styles.jsx
------index.jsx
```

Where `index.jsx` re-exports the contents of `HorizontalScrollNav.jsx` to avoid having this kind of imports:

`import { HorizontalScrollNav } from "components/HorizontalScrollNav/HorizontalScrollNav`

Example of index.jsx:

```
export * from "./HorizontalScrollNav"
```

Reply:
I have a different opinion about it. This will fill your file with a lot of unnecessary imports. I suggest to move the styled components to the bottom of the file if you think this is making it hard to read.